pkg/controller/tenancy/tenancy.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2023 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tenancy
+
+import (
+	"github.com/tigera/operator/pkg/common"
+)
+
+func GetWatchNamespaces(multiTenant bool, defaultNS string) (installNS, truthNS string, watchNamespaces []string) {
```

Comment:
I think I addressed this duplication slightly differently in my PR, but largely the same spirit. Let's keep this and I'll reconcile them when I rebase my PR